openxr/examples/vulkan.rs

```diff
@@ -0,0 +1,649 @@
+                        format: COLOR_FORMAT.as_raw() as _,
+                        sample_count: 1,
+                        width: resolution.width,
+                        height: resolution.height,
+                        face_count: 1,
+                        array_size: VIEW_COUNT,
+                        mip_count: 1,
+                    })
+                    .unwrap();
+                let images = handle.enumerate_images().unwrap();
+                swapchain = Some(Swapchain {
+                    handle,
+                    resolution,
+                    buffers: images
+                        .into_iter()
+                        .map(|color_image| {
+                            let color_image = vk::Image::from_raw(color_image);
+                            let color = vk_device
+                                .create_image_view(
+                                    &vk::ImageViewCreateInfo::builder()
+                                        .image(color_image)
+                                        .view_type(vk::ImageViewType::TYPE_2D)
+                                        .format(COLOR_FORMAT)
+                                        .subresource_range(vk::ImageSubresourceRange {
+                                            aspect_mask: vk::ImageAspectFlags::COLOR,
+                                            base_mip_level: 0,
+                                            level_count: 1,
+                                            base_array_layer: 0,
+                                            layer_count: 2,
+                                        }),
+                                    None,
+                                )
+                                .unwrap();
+                            let framebuffer = vk_device
+                                .create_framebuffer(
+                                    &vk::FramebufferCreateInfo::builder()
+                                        .render_pass(render_pass)
+                                        .width(resolution.width)
+                                        .height(resolution.height)
+                                        .attachments(&[color])
+                                        .layers(1), // Multiview handles addressing multiple layers
+                                    None,
+                                )
+                                .unwrap();
+                            Framebuffer { framebuffer, color }
+                        })
+                        .collect(),
+                });
+            }
+            let swapchain = swapchain.as_mut().unwrap();
+            let image_index = swapchain.handle.acquire_image().unwrap();
+            swapchain.handle.wait_image(xr::Duration::INFINITE).unwrap();
+
+            // Ensure the last use of this frame's resources is 100% done
+            vk_device
+                .wait_for_fences(&[fences[frame]], true, !0)
```

Comment:
Probably nitpicking since !0 should be clear enough, but maybe using an explicit max value would be even clearer.

Reply:
Good thought.